src/httprpc.cpp

```diff
@@ -223,6 +248,17 @@ static bool InitRPCAuthentication()
         LogPrintf("Config options rpcuser and rpcpassword will soon be deprecated. Locally-run instances may remove rpcuser to use cookie-based auth, or may be replaced with rpcauth. Please see share/rpcuser for rpcauth auth generation.\n");
         strRPCUserColonPass = gArgs.GetArg("-rpcuser", "") + ":" + gArgs.GetArg("-rpcpassword", "");
     }
+
+    for (const std::string& strRPCWhitelist : gArgs.GetArgs("-rpcwhitelist")) {
+        auto pos = strRPCWhitelist.find(':');
+        std::string strUser = strRPCWhitelist.substr(0, pos);
+        std::set<std::string>& whitelist = rpc_whitelist[strUser];
+        if (pos != std::string::npos) {
```

Comment:
Should this maybe error? It now silently ignores `nocolonvalue`s.

Reply:
It actually doesn't silently ignore it -- it sets an empty whitelist.